README.md

```diff
@@ -0,0 +1,200 @@
+# Global CO2 from Cement Production Dataset
+
+This repository downloads the Andrew dataset on global CO2 emissions from cement production from [Zenodo](https://zenodo.org/records/10008931).
+
+## Description
+
+This repository downloads data on global CO2 emissions from cement production from [Zenodo](https://zenodo.org/records/10008931).
+The downloaded dataset can then be converted into CSV (.csv file extension) or NetCDF (.nc file extension) format.
```

Comment:
```suggestion
The downloaded dataset can then be converted into CSV (.csv file extension) or NetCDF (.nc file extension) format. Converted data are available for the following versions:
| v231016 |[Zenodo](https://zenodo.org/records/10008931) |
| v230913 |[Zenodo](https://zenodo.org/records/8339353) |
| v230428 |[Zenodo](https://zenodo.org/records/7875557) |
| v220915 |[Zenodo](https://zenodo.org/records/7081360) |
| v220516 |[Zenodo](https://zenodo.org/records/6553090) |
```

Reply:
I think it's good to have the versions that are available somewhere. And the info that you don't need to convert anything. The table is not displayed properly by github. Not sure if it will work in the README.